packages/@functionless/asl-graph/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@functionless/asl-graph",
```

Comment:
is `asl-graph` the whole transpiler? (I have been just calling the intermediate AST for ASL `asl-graph`, and the transpiler uses it, fine with whatever, just want to understand the thought)

Reply:
It's both ASLGraph and the ASL transpiler. Think I should rename it to `asl`?

Reply:
Yeah, I'd like to pull asl-graph out someday as it's own sub-ast and typesafe ASL builder.